module-2/consts/index.ts

```diff
@@ -7,3 +7,5 @@ export const MAX_AGE = 130;
 
```

Comment:
FYI
This is highly subjective, however such files are kept as "constants.ts" alongside with "types", "config" etc directories under "core" directory.
It's just more likely to see the "core/constants.ts", "core/types/*.ts", "core/config/*.json" as a part of the project structure.
The changes up to you.

Reply:
Applied the changes, thanks for suggestion